.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Ansible lint
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    env:
+      ANSIBLE_FORCE_COLOR: 'true'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7]
```

Comment:
What's the reasoning behind running this with two different python versions?

Reply:
copy paste from pulp.. :D